src/main/java/org/jitsi/videobridge/octo/ConfOctoTransport.java

```diff
@@ -206,6 +206,11 @@ public void setRelays(Collection<String> relays)
         setTargets(socketAddresses);
     }
 
+    public String getRelayId()
```

Comment:
Looks like this isn't used anywhere?  (Maybe this is part of what you're planning on pulling out)

Reply:
It's used in kotlin as `relayId` (I had the same thought)